modules/RTC/RTC.js

```diff
@@ -486,7 +486,7 @@ export default class RTC extends Listenable {
             delete this.remoteTracks[
                 toBeRemoved.getParticipantId()][toBeRemoved.getType()];
 
-            this.rtc.eventEmitter.emit(
```

Comment:
it looks like it was changed a couple months ago. I was seeing errors that the object was undefined (since this.rtc is naturally not defined in the RTC module).